cmd/ingester/main.go

```diff
@@ -33,6 +34,13 @@ func main() {
 		ReadTimeout:  5 * time.Second,
 		WriteTimeout: 10 * time.Second,
 		IdleTimeout:  15 * time.Second,
+		TLSConfig: &tls.Config{
```

Comment:
nit: Should we also add `PreferServerCipherSuites` according to the Cloudflare guidelines?

Reply:
There's no need any more - it's a deprecated field.
crypto/tls/common.go line 655 and following:
```
	// PreferServerCipherSuites is a legacy field and has no effect.
	//
	// It used to control whether the server would follow the client's or the
	// server's preference. Servers now select the best mutually supported
	// cipher suite based on logic that takes into account inferred client
	// hardware, server hardware, and security.
	//
	// Deprecated: PreferServerCipherSuites is ignored.
```